src/scenes/gameplay-scene.ts

```diff
@@ -299,7 +299,9 @@ export class GameplayScene {
         if (distance <= 40) {
           this.pickedStoneObject = sc;
           this.pickedStone = sc;
-          this.audioPlayer.playAudio(AUDIO_PATH_ON_DRAG); // Note: When refactoring, this should be moved alongside the StoneHandler and handled within that class.
+          if (this.pickedStone.frame > 99) {
+            this.audioPlayer.playAudio(AUDIO_PATH_ON_DRAG); // Note: When refactoring, this should be moved alongside the StoneHandler and handled within that class.
+          }
```

Comment:
Instead of adding note, why not just moved it to Stonehandler and create a method that will play the audio. You have access to `this.stoneHandler` so I think you can call that method. That way no need to write a note on what to do as you have already moved it.

Reply:
@bernhardccodev moved this logic to stonehandler. thank you for your review Berns